app/src/main/java/com/example/egida/domain/useCase/scoreBall/UseCaseScoreBal.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.egida.domain.useCase.scoreBall
+
+import com.example.egida.domain.entity.UserDatabase
+import kotlinx.coroutines.flow.Flow
+
+interface UseCaseScoreBal {
```

Comment:
XxxYyyUseCase and XxxYyyUseCaseImpl - better naming. Also - "Bal" - could be replaced with more telling word.